Bugsnag/Helpers/BSGFeatureFlagStore.m

```diff
@@ -57,10 +62,150 @@ void BSGFeatureFlagStoreClear(BSGFeatureFlagStore *store, NSString *_Nullable na
                     if (![variant isKindOfClass:[NSString class]]) {
                         variant = nil;
                     }
-                    [store addObject:[BugsnagFeatureFlag flagWithName:featureFlag variant:variant]];
+                    [store addFeatureFlag:featureFlag withVariant:variant];
                 }
             }
         }
     }
     return store;
 }
+
+- (_Nonnull instancetype) init {
+    if ((self = [super init]) != nil) {
+        _flags = [NSMutableArray new];
+        _indices = [NSMutableDictionary new];
+    }
+    return self;
+}
+
+static inline int getIndexFromDict(NSDictionary *dict, NSString *name) {
+    NSNumber *boxedIndex = dict[name];
+    if (boxedIndex == nil) {
+        return -1;
+    }
+    return boxedIndex.intValue;
+}
+
+- (BOOL)isEqual:(nullable id)object {
```

Comment:
It's called elsewhere in the code

Reply:
It's only used by `BSGFeatureFlagStoreTests`

It makes no sense to bloat the shipping framework with code only used by unit tests, so it would be better to remove this 👍